swahili/tembelea-jw-nukta-org/tembelea-jw-nukta-org.md

```diff
@@ -0,0 +1 @@
+Ulipotembelea tuvuti hiyo ya [jw nukta org](www.jw.org/sw) ukapata majibu yaliyokufariji ya maswali muhimu maishani.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.